---
title: Extracting DNA from Plants for DArT
layout: article
keywords: ["plants", "dna"]
description: "DArTFAQ."
author: DArTSupport
author_image: /assets/img/author.png
date: August 11, 2022
updated: August 01, 2022
categories: ordering
---

## Targeted Genotyping Specific Requirements

**_(Copy for this page to be confirmed after review relating to updated DNA extraction_**
**_processes. This interim copy as provided.)_**

We applied **DArTseq**, and more recently DArTseqLD, to many hundreds of organisms
for a wide range of applications, especially in genetic diversity characterisation and
marker discovery area.

However, there are many applications which demand genotyping using selected marker
set rather than randomly distributed markers reported by DArTseq/DArTseqLD methods.
We have developed three technologies to offer the capacity to genotype materials with
specific markers (mainly SNPs). These technologies vary in the marker density capacity
as well as the initial cost of development, therefore we select the most appropriate
technology for specific clients’ needs. Because the assays are always developed for
specific material and marker sets we do not offer them to general public- you will not
see them in our online ordering system unless you are representing the client for whom
the assay was developed. Please contact us if you want to discuss your needs for
specific marker characterisation.

Thanks to a number of technologies, we have developed we can offer a very cost
effective solution to practically any application that requires anything from as few as a
dozen of markers up to at least 10,000 markers.

### DArTcap

DArTcap is used in similar applications as DArTseqLD, but it applies a selective step
after complexity reduction to genotype specific markers from DArTseq representations.
This selection is achieved with the use of the nucleic acid “capture probes” that bind to
restriction fragments in the representations carrying the specific DArTseq markers. The
restriction fragments are highly enriched in this capture step and most of the sequences
obtained from the sequencing of the captured fragments represent the targeted
markers. DArTcap is fairly flexible in terms of the number of markers that can be
selected, but in most cases we target from a few hundreds to a few thousands markers.
DArTcap is particularly useful in polyploids as the initial complexity reduction step
(DArTseq library construction) prior to capture step would have already selected only
one of the homeologous sequences at most loci. This feature enables highly effective
sub-genome specific genotyping and obtaining co-dominant SNPs even in highly
polyploid species with little or no genome sequence information.
Back to top Shortcut to ‘ecology’ menu 80

### DArTag

DArTag method is targeting similar number of markers as DArTcap (many hundreds to thousands), but is not restricted to markers identified by DArTseq platform. Any SNP (or a small indel) can be targeted using DArTag if there is some genomic sequence available around the variant base/indel. In the first step of DArTag special molecular probes select the small target regions containing sequence variants. In the second step the targeted regions are amplified and, in parallel, the sample-specific barcode (a short stretch of sequence) is attached. The libraries generated in that manner are sequenced on the NGS equipment and resulting sequences processed using DArT PL’s proprietary pipeline

### DArTmp

DArTmp is used in applications requiring relatively small number of markers, but it can effectively process a few hundred SNPs in a single assay. There is no minimal number of markers that are required for the assay and DArT P/L has been delivering services with as few as 10 markers in one assay. However, the advantage of DArTmp compared to any single marker assay platforms is its ability to be increasingly fast as the loci it targets increases. The DArTmp assay is very similar in terms of requirements and the process to the DArTag assay as it also targets a short stretch of sequence around the sequence variant of interest. The main difference is the way the selection of the target sequence happens in the first step of the assay. In DArTmp a pair of oligonucleotide primers are used instead of a single probe applied in DArTag. All the consecutive steps in the lab and the whole analytical part are the same for DArTmp and DArTag. As DArTmp is targeting the smallest number of markers, the initial investment in assay development and oligo synthesis is small, enabling rapid adoption of this method to a number of applications requiring only modest number of markers.

**Targeted Genotyping (DArTag, DArTmp)**

- DArTmp and DArTag services are processed in 384 well PCR plate, thus we require the services to be ordered in multiples of four 94 well PCR plates (wells G12 and H12 must be empty for our internal controls).
- We can process partial 384 well plates, however the cost per sample will increase: the less samples the higher cost.
- Both services require the submission of a positive control (PC).
  - Minimum 500µl of ~25ng/µl of good quality gDNA in EB buffer (10mM Tris-Cl, pH8.5) shipped in a microtube
  - We recommend checking for presence of residual nucleases in your DNA prep as it may degrade the gDNA during transportation. Follow instructions in the FAQ “How can I check DNA quality for DArT assays?” https://www.diversityarrays.com/faq/
  - PC can be shipped as precipitated in Ethanol or as dry pellet. Please indicate DNA storage method and gDNA quantity
  - Alternatively, we can extract gDNA for PC in our facilities. Submit enough tissue to obtain 10-12µg of gDNA
  - Please include the printed copy of Positive Control Specification document (download below) and all other required documentation depending what (tissue/gDNA) and from where (domestic/international) is shipped

Positive Control (gDNA) Specification for Targeted Genotyping Services

Positive Control (tissue) Specification for Targeted Genotyping Services

If you require any additional information about the PC submission, please contact us at samples@diversityarrays.com.
